raiden_contracts/constants.py

```diff
@@ -97,30 +97,14 @@ class MessageTypeId(IntEnum):
 # Network configurations
 START_QUERY_BLOCK_KEY = 'DefaultStartBlock'
 
-
-class ChainId(IntEnum):
-    MAINNET = 1
-    ROPSTEN = 3
-    RINKEBY = 4
-    KOVAN = 42
-    SMOKETEST = 627
-
-
-MAINNET = 'mainnet'
-ROPSTEN = 'ropsten'
-RINKEBY = 'rinkeby'
-KOVAN = 'kovan'
-SMOKETEST = 'smoketest'
-
 ID_TO_NETWORKNAME = {
-    ChainId.MAINNET: MAINNET,
-    ChainId.ROPSTEN: ROPSTEN,
-    ChainId.RINKEBY: RINKEBY,
-    ChainId.KOVAN: KOVAN,
-    ChainId.SMOKETEST: SMOKETEST,
+    1: 'mainnet',
```

Comment:
But can you make them also constants?

e.g.:

NETWORKID_MAINNET = 1
NETWORKID_ROPSTEN = 3
...
```

e.t.c.

Reply:
It would allow us in the client to refer to ropsten's (or any other network's) ID programmatically without going through the `ID_TO_NETWORKNAME` whenever we want network specific behavior. It's just a convenience to avoid a mapping lookup.

And constant variables are not redundant in constants.py :)

Reply:
> And constant variables are not redundant in constants.py :)

My problem is defining constants in `raiden-contracts` that are not used in `raiden-contracts`. If they are used **only** in `raiden`, then `raiden` should define them. Otherwise, we can break things by mistake.
To be honest, I would not even use `raiden_contracts.ID_TO_NETWORKNAME` in `raiden`... because of this. I would only stick to the `raiden_contracts` specific deliverables: contracts source, compiled files, `ContractManager` and the specific contracts-related constants.

But if you really want them added, I will.

Reply:
By the way, the entire `ID_TO_NETWORK_CONFIG` will be removed in the PR that solves https://github.com/raiden-network/raiden-contracts/issues/321
We will have `json` files with deployment data. So, any `raiden` specific network configs will be handled by `raiden` anyway

Reply:
> My problem is defining constants in raiden-contracts that are not used in raiden-contracts.

Last time we discussed I had understood you wanted contract deployment related data to be in contracts. It used to be in raiden constants.py but then got moved here.

I really have no strong opinion on this. Just finalize these as quick as possible so that the client code has time to adapt. Whatever you remove from here, we will have in the client.

> The contracts will just provide a way to get deployment info for a specific version & chain.

So will the contracts package provide a method for us to read this json file or will we have to implement it in the client?

Reply:
>  a method for reading the json file based on version (default=current version) and the chain id, to make it easier for clients. Does this sound ok?

I think it does. These 2 pieces of information is what the client will have. At the moment we also have this `development | release` configuration as additional data. I guess we can incorporate that in the version?

Reply:
> development | release

Not sure what this is about. Are you talking about the pre-limits version and current version of the contracts? For this, I will have 2 versioned `data` dirs with the compiled + deployment info. E.g. `data_0.4.1` for current one, and `data_[PRE_LIMITS_VERSION]` (I'll check what version that was)